models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -240,11 +240,19 @@ public function isOptimizedAdminLoading(): bool
         return true;
     }
 
+    /**
+     * @param DataObject\Concrete $container
+     * @param array $params
+     * @return array
+     */
```

Comment:
```suggestion
    /**
     * {@inheritdoc}
     */
```
please change `ManyToManyObjectRelation` phpdoc as well otherwise it will throw declaration error on adding typehints.